apis/python/tools/ingestor

```diff
@@ -135,25 +135,32 @@ def ingest_one(input_path: str, output_path: str, ifexists: str, soma_options: t
         if not os.path.exists(parent):
             os.mkdir(parent)
 
-    if os.path.exists(output_path):
-        if ifexists == "continue":
-            print(f"Already exists; continuing: {output_path}")
-            return
-        elif ifexists == "abort":
-            print(f"Already exists; aborting: {output_path}")
-            sys.exit(1)
-        elif ifexists == "replace":
-            print(f"Already exists; replacing: {output_path}")
-            shutil.rmtree(output_path)  # Overwrite
-        else:
-            raise Exception(
-                "Internal coding error in --ifexists handling.", ifexists, "<"
-            )
-
     soma = tiledbsc.SOMA(uri=output_path, soma_options=soma_options)
 
-    # Do the ingest into TileDB.
-    tiledbsc.io.from_h5ad(soma, input_path)
+    if ifexists == "update_obs_and_var":
+        if not os.path.exists(output_path):
+            raise Exception("Cannot update; doesn't exist yet:", output_path)
+        # Do the ingest into TileDB.
+        tiledbsc.io.from_h5ad_update_obs_and_var(soma, input_path)
+
+    else:
+        if os.path.exists(output_path):
+            if ifexists == "continue":
+                print(f"Already exists; continuing: {output_path}")
+                return
+            elif ifexists == "abort":
+                print(f"Already exists; aborting: {output_path}")
+                sys.exit(1)
+            elif ifexists == "replace":
+                print(f"Already exists; replacing: {output_path}")
+                shutil.rmtree(output_path)  # Overwrite
```

Comment:
This only works for local files. Maybe a nice error if ingesting to s3://, tiledb://, etc.